src/functions/validateTime.js

```diff
@@ -30,10 +54,12 @@ export const validateTime = (bookings, date, start, end) => {
     alert("You can't book more than 60 days in advance.");
     return false;
   } else {
-    for (let i = 0; i < bookings.length; ++i) {
+    const dissectedBookings = dissectBookings([...bookings]);
```

Comment:
`[...bookings]` ? just do `dissectBookings(bookings)` cuz `bookings` is an array right?

Reply:
Oh yaa, I did that on purpose actually. Since array is passed by reference as a function argument, `[...bookings]` creates a new copy of `bookings` and won't effect the original `bookings` array. Just to be safe, in case there is any data changing operation in `dissectBookings` tbh 😂